systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java

```diff
@@ -321,4 +322,53 @@ public static String getConnectorStatuses(AddressSpace addressSpace) {
                 .toString();
     }
 
+    public static Endpoint getMessagingRoute(AddressSpace addressSpace) throws Exception {
+        Endpoint messagingEndpoint = AddressSpaceUtils.getEndpointByServiceName(addressSpace, "messaging");
+        if (messagingEndpoint == null) {
+            String externalEndpointName = AddressSpaceUtils.getExternalEndpointName(addressSpace, "messaging-" + AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace));
+            messagingEndpoint = Kubernetes.getInstance().getExternalEndpoint(externalEndpointName);
+        }
+        if (TestUtils.resolvable(messagingEndpoint)) {
+            return messagingEndpoint;
+        } else {
+            return Kubernetes.getInstance().getEndpoint("messaging-" + AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace), addressSpace.getMetadata().getNamespace(), "amqps");
+        }
+    }
+
+    public static String getOCConsoleRoute() {
```

Comment:
this doesn't belong here, it fits better in kubernetes or openshift class or some other utils class

Reply:
yup you are right